neurodsp/spectral/power.py

```diff
@@ -136,6 +139,50 @@ def compute_spectrum_wavelet(sig, fs, freqs, avg_type='mean', **kwargs):
     return freqs, spectrum
 
 
+@multidim(select=[0])
+def compute_spectrum_fft(sig, fs, window=None, nfft=None, f_range=None):
```

Comment:
could be added to \_\_init__.py so it can be directly from neurodsp.spectral like the other compute_spectrum_* functions

Reply:
Yeh, good call - added!